src/server/game/Battlefield/Zones/BattlefieldWG.h

```diff
@@ -34,46 +35,81 @@ float const WintergraspStalkerPos[4]    = { 4948.985f, 2937.789f, 550.5172f, 1.8
 enum WintergraspSpells
 {
     // Wartime auras
-    SPELL_RECRUIT                               = 37795,
-    SPELL_CORPORAL                              = 33280,
-    SPELL_LIEUTENANT                            = 55629,
-    SPELL_TENACITY                              = 58549,
-    SPELL_TENACITY_VEHICLE                      = 59911,
-    SPELL_TOWER_CONTROL                         = 62064,
-    SPELL_SPIRITUAL_IMMUNITY                    = 58729,
-    SPELL_GREAT_HONOR                           = 58555,
-    SPELL_GREATER_HONOR                         = 58556,
-    SPELL_GREATEST_HONOR                        = 58557,
-    SPELL_ALLIANCE_FLAG                         = 14268,
-    SPELL_HORDE_FLAG                            = 14267,
-    SPELL_GRAB_PASSENGER                        = 61178,
+    SPELL_RECRUIT,
+    SPELL_CORPORAL,
+    SPELL_LIEUTENANT,
+    SPELL_TENACITY,
+    SPELL_TENACITY_VEHICLE,
+    SPELL_TOWER_CONTROL,
+    SPELL_SPIRITUAL_IMMUNITY,
+    SPELL_GREAT_HONOR,
+    SPELL_GREATER_HONOR,
+    SPELL_GREATEST_HONOR,
+    SPELL_ALLIANCE_FLAG,
+    SPELL_HORDE_FLAG,
+    SPELL_GRAB_PASSENGER,
 
     // Reward spells
-    SPELL_VICTORY_REWARD                        = 56902,
-    SPELL_DEFEAT_REWARD                         = 58494,
-    SPELL_DAMAGED_TOWER                         = 59135,
-    SPELL_DESTROYED_TOWER                       = 59136,
-    SPELL_DAMAGED_BUILDING                      = 59201,
-    SPELL_INTACT_BUILDING                       = 59203,
+    SPELL_VICTORY_REWARD,
+    SPELL_DEFEAT_REWARD,
+    SPELL_DAMAGED_TOWER,
+    SPELL_DESTROYED_TOWER,
+    SPELL_DAMAGED_BUILDING,
+    SPELL_INTACT_BUILDING,
 
-    SPELL_TELEPORT_BRIDGE                       = 59096,
-    SPELL_TELEPORT_FORTRESS                     = 60035,
+    SPELL_TELEPORT_BRIDGE,
+    SPELL_TELEPORT_FORTRESS,
 
-    SPELL_TELEPORT_DALARAN                      = 53360,
-    SPELL_VICTORY_AURA                          = 60044,
+    SPELL_TELEPORT_DALARAN,
+    SPELL_VICTORY_AURA,
 
     // Other spells
-    SPELL_WINTERGRASP_WATER                     = 36444,
-    SPELL_ESSENCE_OF_WINTERGRASP                = 58045,
-    SPELL_WINTERGRASP_RESTRICTED_FLIGHT_AREA    = 58730,
-    SPELL_LEANING_TOWER_ACHIEVEMENT             = 60676,
+    SPELL_WINTERGRASP_WATER,
+    SPELL_ESSENCE_OF_WINTERGRASP,
+    SPELL_WINTERGRASP_RESTRICTED_FLIGHT_AREA,
+    SPELL_LEANING_TOWER_ACHIEVEMENT,
 
     // Phasing spells
-    SPELL_HORDE_CONTROLS_FACTORY_PHASE_SHIFT    = 56618,// ADDS PHASE 16
-    SPELL_ALLIANCE_CONTROLS_FACTORY_PHASE_SHIFT = 56617,// ADDS PHASE 32
+    SPELL_HORDE_CONTROLS_FACTORY_PHASE_SHIFT,
+    SPELL_ALLIANCE_CONTROLS_FACTORY_PHASE_SHIFT,
 
-    SPELL_HORDE_CONTROL_PHASE_SHIFT             = 55773,// ADDS PHASE 64
-    SPELL_ALLIANCE_CONTROL_PHASE_SHIFT          = 55774,// ADDS PHASE 128
+    SPELL_HORDE_CONTROL_PHASE_SHIFT,
+    SPELL_ALLIANCE_CONTROL_PHASE_SHIFT,
```

Comment:
will this be tied to the order of declaration of the `WG_SPELLS` below? I'm not sure about it, because it can easily lead to mistakes